apps/frontend/src/config.ts

```diff
@@ -7,6 +7,14 @@ export const fontBase = Inter({ subsets: ["latin"], variable: "--font-base" });
 export const BASE_API_URL =
   process.env.NEXT_PUBLIC_API_URL || "http://localhost:8080/api";
 
+export const OAUTH_APP_MAPPING: Record<string, {id: string | undefined, secret: string | undefined, token_url: string }> = {
+  "strava": {
+    token_url: "https://www.strava.com/api/v3/oauth/token",
+    id: process.env.NEXT_PUBLIC_OAUTH_STRAVA_CLIENT_ID || "",
+    secret: process.env.NEXT_PUBLIC_OAUTH_STRAVA_CLIENT_SECRET || "",
+  }
+}
+
```

Comment:
Can this live in config or should it live in constants.ts? Put it here to be with other env vars.